modules/openshift-developer-cli-installing-odo-on-linux.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/installing-odo.adoc     
+
+[id="installing-odo-on-linux"]
+
+= Installing {odo-title} on Linux
+
+== RPM installation
+* For CentOS or RHEL use:
++
+----
+$ yum install odo
+----
+* For Fedora use:
++
+----
+$ dnf install odo
+----
+
```

Comment:
This above section should be removed. It was a requirement for our release procress, but there are **barely** any users which are running RHEL as a desktop machine with OpenShift entitlement enabled. This should be removed.

Reply:
Ping @mohammedzee1000 @kadel 

I discussed this with @kadel and this should be 100% removed as similarly to `oc`, `odo` should be installed via binary.